deploy/apecloud-mysql-cluster/templates/_helpers.tpl

```diff
@@ -35,28 +10,41 @@ Common labels
 */}}
 {{- define "apecloud-mysql-cluster.labels" -}}
 helm.sh/chart: {{ include "apecloud-mysql-cluster.chart" . }}
-{{ include "apecloud-mysql-cluster.selectorLabels" . }}
+{{ include "cluster-libchart.clusterLabels" . }}
 {{- if .Chart.AppVersion }}
 app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
 {{- end }}
 app.kubernetes.io/managed-by: {{ .Release.Service }}
 {{- end }}
 
 {{/*
-Selector labels
+Define the cluster componnets with proxy
+TODO: calculate the proxy resoruces based on the mysql resources
 */}}
-{{- define "apecloud-mysql-cluster.selectorLabels" -}}
-app.kubernetes.io/name: {{ include "apecloud-mysql-cluster.name" . }}
-app.kubernetes.io/instance: {{ .Release.Name }}
+{{- define "apecloud-mysql-cluster.proxyComponents" }}
```

Comment:
following components are defined in mysql-scale CD, will 2 CDs be merged?

Reply:
Now, this chart use different CD according to proxy enabled or not. IMO, we should merge these two CDs to one, what do you think?

Reply:
These two CDs will be merged after cli can create cluster with more components correctly.

Reply:
> These two CDs will be merged after cli can create cluster with more components correctly.

"will be merge" as it's work in progress?

Reply:
https://github.com/apecloud/kubeblocks/pull/3995